src/main/java/org/folio/converter/StringConstants.java

```diff
@@ -0,0 +1,53 @@
+package org.folio.converter;
+
+public class StringConstants {
```

Comment:
It is good practice to keep constatns class non-extendable by making it "final", and hiding constructor by private definition
`private StringConstants(){}`